regolith/helpers/Google_Calendar_API/google_calendar_api.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import print_function
```

Comment:
I think this stuff should also be in tools and no Google_Calendar_API (and therefore no modules in there), so as not to change the API of regolith for some small side-case of Google calendar functionality.